templates/terraform/command-center-project/service-accounts.tf

```diff
@@ -10,3 +10,27 @@ module command_center_gke_runner_account {
   vault_path = "${local.vault_prefix}/service-accounts/gke-runner"
   roles = ["logging.logWriter", "monitoring.metricWriter", "monitoring.viewer"]
 }
+
+module clinvar_argo_runner_account {
+  source = "/templates/google-sa"
+  providers = {
+    google.target = google.target,
+    vault.target = vault.target
+  }
+
+  account_id = "clinvar-argo-runner"
+  display_name = "Service account to run ClinVar's Argo workflow."
+  vault_path = "${local.vault_prefix}/service-accounts/clinvar-argo-runner"
+  roles = ["container.developer"]
+}
+
+data google_project current_project {
+  provider = google.target
+}
```

Comment:
I'm not sure if there is a better place to put this data object. Let me know if you have any suggestions!